### DISCUSSION TITLE
Datum im DP

### DISCUSSION THREAD

**Discussion by Mephisto090** [mcuiobroker/docuJarvis#8] [Q&A]
Hej,

bevor ich Zerfu damit belästige. Gibt es eine Möglichkeit schon beim Gerät ein Datum zu formatieren oder geht das nur via js/blocky?

So sieht es jetzt aus: 2022-10-10T19:47:13.181Z

![image](https://user-images.githubusercontent.com/85995176/194925102-cf3d9b86-72d3-4b95-9f35-97dad868bf57.png)
![image](https://user-images.githubusercontent.com/85995176/194925140-a2a56748-3606-4e21-874b-cc393ff3fa3b.png)

**Comment by mcuiobroker** [comment]
Das müsste in Datenpunkt-Eigenschaften gehen.
Woher kommt das Datum, wer erstellt es?
Er würde es in 10.10.2022 21:47 wandeln? Was ja anscheinend nicht stimmen würde.
wenn das Datum 10.10.2022 19:47 sein soll, dann hätte der UTC-String
2022-10-10T19:47:13.181+02:00 so aussehen müssen.
Ursache kann sein, dass auf dem Erstellungssystem die Zeit nicht korrekt eingestellt ist. Und er deswegen, die Zeit falsch übermittelt.

EDIT: Doch falsches Datum. Ich hatte für die Tests das Datum schon geändert.
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/geraet/datenpunkt-eigenschaft/utc-gmt-datum

**Reply by Mephisto090** [reply]
Das kommt von deConz mit einem Sonoff Temperaturfühler. Ich teste gleich mal

**Reply by mcuiobroker** [reply]
https://github.com/Jey-Cee/ioBroker.deconz/issues/63
